### PR TITLE
Feature/add services

### PR DESCRIPTION
**Pull request by germanilia**
Issue Number: N/A

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/code_of_conduct.md) file for detailed contributing guidelines.

### CODE REVIEW

File: helm/charts/services/amplication-server/values.yaml [hunk 1]
```diff
@@ -44,3 +43,39 @@ postgres:
...
+#     enabled: false
+#   cp-kafka-rest:
+#     enabled: false
+#   cp-kafka-connect:
+#     enabled: false
+#   cp-control-center:
+#     enabled: false
+#   cp-kspl-server:
+#     enabled: false
+
```
Comment (mshidlov) [inline helm/charts/services/amplication-server/values.yaml:81]: @germanilia  can we remove it??
```suggestion 
```

File: helm/charts/services/amplication-server/Chart.yaml [hunk 2]
```diff
@@ -10,6 +10,11 @@ dependencies:
   repository: "https://charts.bitnami.com/bitnami"
   condition: postgres.install
 
+# - name: cp-helm-charts
+#   version: "0.6.1"
+#   repository: "https://confluentinc.github.io/cp-helm-charts/"
+#   condition: cp-helm-charts.cp-kafka.enabled
+
```
Comment (mshidlov) [inline helm/charts/services/amplication-server/Chart.yaml:17]: @germanilia Do we need it?
```suggestion
```

File: helm/charts/services/amplication-git-push-webhook-service/values.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  type: ClusterIP
+  port: 
+    target: 3000
+  protocol: TCP
+
+ingress:
+  enabled: true
+  hostname: https://webhook.amplication.com
+
+pvc:
```
Comment (mshidlov) [inline helm/charts/services/amplication-git-push-webhook-service/values.yaml:29]: @germanilia this service doesn't need PVC

File: helm/charts/services/amplication-server/values.yaml [hunk 4]
```diff
@@ -7,15 +7,16 @@ autoscaling:
...
-volume:
-  name: amplication-server-artifacts-data
-  path: /artifacts
 image:
   pullPolicy: Always
   repository: 407256539111.dkr.ecr.us-east-1.amazonaws.com/amplication-prod-server
   tag: 0.0.7
   imagePullSecrets: []
 
+volume:
```
Comment (mshidlov) [inline helm/charts/services/amplication-server/values.yaml:16]: @tupe12334 @abrl91 this should be in all services that require EFS

File: helm/charts/services/amplication-server/values.yaml [hunk 5]
```diff
@@ -7,15 +7,16 @@ autoscaling:
...
+  name: efs-claim
+  path: /etc/amplication-server
```
Comment (mshidlov) [inline helm/charts/services/amplication-server/values.yaml:18]: /amplication-data

File: helm/charts/amplication/values-staging-os.yaml [hunk 6]
```diff
@@ -50,4 +55,65 @@ amplication-server:
...
+      #POSTGRESQL_URL: postgres://admin:admin@localhost:5432
+
+      #EXISTS SECRETS
+      #GITHUB_APP_PRIVATE_KEY:
+      #GITHUB_APP_CLIENT_SECRET:
+
+      NODE_ENV: "production"
+
+      #EFS
+      STORAGE_PATH: [EFS-MOUNT-PATH]/git-data #ILIA
```
Comment (mshidlov) [inline helm/charts/amplication/values-staging-os.yaml:94]: /amplication-data/git-data

[EFS-MOUNT-PATH] = amplication-data


(volume path)

**Comment by mshidlov** [comment]
@abrl91 @tupe12334 please make sure all environments from [141d469](https://github.com/amplication/amplication/pull/2804/commits/141d4697f331d3a818b255390ade9a03ef95225d) is in the last helm version

File: helm/charts/amplication/values-staging-os.yaml [hunk 7]
```diff
@@ -23,11 +23,14 @@ amplication-server:
       HOST: "https://server.staging-os.amplication-dev.com"
```
Comment (abrl91) [inline helm/charts/amplication/values-staging-os.yaml:23]: I am unsure if we need the '-dev' here @mshidlov. You asked to compare this commit to the last one in this pull request, and it is with the '-dev' here and in the other commits.

**Pull request merged**